flint/prefect/flows/continuum_pipeline.py

```diff
@@ -251,98 +253,104 @@ def process_science_fields(
     fits_beam_masks = None
 
     for current_round in range(1, field_options.rounds + 1):
```

Comment:
Much of this has changed simply from the `with tags` context manager. But do note passing in the field summary to the appropriate linmos tasks